addons/website_mail_group/static/src/snippets/s_group/000.js

```diff
@@ -28,8 +28,6 @@ MailGroup.include({
             return;
         }
 
-        this.$el.removeClass('d-none');
```

Comment:
@can-odoo If you do this, it means that all existing snippets in customer databases will suddenly stop appearing...

Reply:
Hello @qsm-odoo, sorry but didn't get your point here.
Here we are handling`s_group` snippet only, how it will affect the existing snippets to suddenly stop appearing??

Reply:
Try it and you will see:

- Add a s_group in your page on a standard 15.0 instance
- Add your commit and reload your server
=> Your snippet does not appear anymore

Reply:
Woah...okay gotcha 👍🏽

Reply:
@qsm-odoo
>But add a comment next the d-none class addition in the start method of the public widget

Comment added, let me know if needs more/modification